src/rest_framework_jwt/blacklist/serializers.py

```diff
@@ -0,0 +1,57 @@
+
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import jwt
+
+from django.contrib.auth import get_user_model
+from django.utils.translation import ugettext as _
```

Comment:
`ugettext` has entered a deprecation period and will raise deprecation warnings. I suggest handling this via the `compat` module.